### PR TITLE
Movies page

### PR DESCRIPTION
**Pull request by Vadim-Bykov**
Created MoviesStackNavigator with two screens. The first one containes three sections  with movies (Popular, Now Playing, Coming soon), different genres, movie pager and short information for the shown movie. Clicking on the section a request is made for corresponding films. The active section is marked witn a dash.
image

image

Clicking on the genre a request is made for corresponding films. The active genre is marked witn a different color.
image

Clicking on the movie picture the MovieScreen is changed to DetailsScreen. Two requests for details and cast of the film are fullfilled.
image

image

Axios and react-query are used to query the backend.

### CODE REVIEW

File: App/Screens/Home/HomeScreenProvider.js [hunk 1]
```diff
@@ -58,15 +68,16 @@ export const HomeScreenProvider = ({navigation}) => {
       );
   }, [genres.isError, movies.isError]);
 
-  const onChangeGenre = async (genreId = 14, page = 1) => {
+  const onChangeGenre = async (genreId = currentGenreID) => {
+    console.log(genrePage);
```
Comment (DzmitrySakalouski) [inline App/Screens/Home/HomeScreenProvider.js]: remove console logs

File: App/Screens/Home/components/Pager/Pager.js [hunk 2]
```diff
@@ -12,40 +12,33 @@ import {useMovieContext} from '../../HomeScreenProvider';
...
             pageMargin={-90}
             offscreenPageLimit={1}
-            style={{
-              height: width * 1.2,
-              width: width * 1,
-              marginBottom: 20,
-              elevation: 15,
-            }}>
+            onPageSelected={e => setActiveIndex(e.nativeEvent.position)}
+            style={{height: width, width: width * 1}}>
```
Comment (DzmitrySakalouski) [inline App/Screens/Home/components/Pager/Pager.js]: move to StyleSheet.create

Comment (DzmitrySakalouski) [reply App/Screens/Home/components/Pager/Pager.js]: width*1

**Review by DzmitrySakalouski — COMMENTED** [review]
Check please comments

File: App/Screens/Home/HomeScreenProvider.js (hunk 1 above)
Comment (Vadim-Bykov) [reply App/Screens/Home/HomeScreenProvider.js]: Removed console.log()

**Pull request merged**